modules/serverless-functions-creating-on-cluster-builds.adoc

```diff
@@ -81,3 +64,27 @@ Please provide credentials for image registry used by Pipeline.
 ----
 
 . To update your function, commit and push new changes by using Git, then run the `kn func deploy --remote` command again.
+
+. Optionally, you can configure your function to be built on the cluster after every Git push by using pipelines-as-code:
+
+.. Generate the Tekton `Pipelines` and `PipelineRuns` configuration for your function:
++
+[source,terminal]
+----
+$ kn func config git set
+----
++
+Apart from generating configuration files, this command connects to the cluster and validates that the pipeline is installed. Using the token, it also creates, on behalf of the user, a webhook on the function repository. That webhook triggers the pipeline on the cluster every time changes are pushed to the repository.
```

Comment:
```suggestion
Apart from generating configuration files, this command connects to the cluster and validates that the pipeline is installed. By using the token, it also creates, on behalf of the user, a webhook on the function repository. That webhook triggers the pipeline on the cluster every time changes are pushed to the repository.
```
IBM Style Word Usage provides this advice about _using_: "To avoid ambiguity, replace this gerund with either _by using_ or _that uses_."